<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8" />
		<title>获取分数</title>
	</head>
	<body>
		<input type="button" name="b1" id="b1" value="加载下一页" />
		<p id="p1"></p>
		
		
		
		<script type="text/javascript">
			var oP1=document.getElementById("p1")
			
			var pageIndex=1;
			var oB1=document.getElementById("b1");
			
//			fn1();//这里不能调用  不然的话   会报错   因为function之后    下面的script已经是在调用
			function fn1(arrdata){
//						arrdata.sort(function(a,b){
//							var oa=parseInt(JSON.parse(a.Data).total);
//							var ob=parseInt(JSON.parse(b.Data).total);
//							return ob-oa;
//						})
					try{
						
						for(var i=0;i<arrdata.length;i++){
						if(arrdata[i].Data){
							var oBj=JSON.parse(arrdata[i].Data)
							oP1.innerHTML+=oBj.name+"&ensp;"+"&ensp;"+"&ensp;"+oBj.total+"<br>";
					}
				}
					}catch(e){
						//TODO handle the exception
					}
				
				
				
			
			}
			var srp=document.createElement("script");
			srp.src="http://10.17.158.224:2801/Product/GetProductsByPage_get?type=chen&pagesize=1000&pageindex=1&callback=fn1"
			document.body.appendChild(srp);

		</script>

		
		
		
	</body>
</html>
